src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  width: 100%;
```

Comment:
You can move this width to header

Reply:
I tried this too, but then I had a problem with my tests